scripts/sct_utils.py

```diff
@@ -137,6 +189,32 @@ def colors(cls):
         return [v for k, v in cls.__dict__.items() if not k.startswith("_") and k is not "colors"]
 
 
+class Config:
+    """
+    Config value use all around the sct
+    """
+    _sentry_config = None
+    _log_dsn = "uggcf://14p24045rq8s4nno97qp86n3o034nn5r:0226q799rss8450qn38oro87rsop904r@fragel.vb/232808"
```

Comment:
I don't agree with having the `_log_dsn` key available to all users. This should be used for internal use.

I would make this an environment variable instead, set it manually.